media/css/mozorg/home/fx-quantum-promo.scss

```diff
@@ -1,62 +0,0 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-@import '../../pebbles/includes/lib';
-
-.quantum-promo {
-    background-color: #45caf3;
-    @include at2x('/media/img/mozorg/promo/quantum-promo-banner-phone.png', 760px, 200px);
```

Comment:
Can delete this image (and the one below) as well?

Reply:
Doh! Good catch.